_studio/mfx_lib/shared/src/mfx_vpx_dec_common.cpp

```diff
@@ -379,7 +379,8 @@ namespace MFX_VPX_Utility
         {
             if (MFX_FOURCC_NV12 != p_in->mfx.FrameInfo.FourCC &&
                 MFX_FOURCC_AYUV != p_in->mfx.FrameInfo.FourCC &&
-                MFX_FOURCC_P010 != p_in->mfx.FrameInfo.FourCC)
+                MFX_FOURCC_P010 != p_in->mfx.FrameInfo.FourCC &&
+                !(MFX_FOURCC_Y410 == p_in->mfx.FrameInfo.FourCC && (hwtype == MFX_HW_ICL || hwtype == MFX_HW_ICL_LP)))
```

Comment:
MFX_FOURCC_Y410 is defined for MFX_VERSION >= 1027 so this check should be enclosed in corresponding #if directives (see l.400-403)